.prettierrc

```diff
@@ -1,6 +1,13 @@
 {
-  "semi": true,
-  "trailingComma": "all",
+  "arrowParens": "avoid",
```

Comment:
My recommendation is to utilize "arrowParens": "always".
Here's the rationale from the Prettier documentation:
"At first glance, avoiding parentheses may look like a better choice because of less visual noise. However, when Prettier removes parentheses, it becomes harder to add type annotations, extra arguments or default values as well as making other changes. Consistent use of parentheses provides a better developer experience when editing real codebases, which justifies the default value for the option."

Reply:
You're right. I must've accidentally overwritten this one with a value from the app repo. I think keeping parentheses is the right move. Thanks for flagging this